### PR TITLE
GitHub ble df add missing features and commands

### PR DESCRIPTION
**Pull request by ppryga-nordic**
The PR adds missing features and commands related with direction finding to list of supported items.

It is required to pass Bluetooth controller qualification tests.

The fixes were verified with EBQ.

### CODE REVIEW

**Review by kruithofa — APPROVED** [review]

File: subsys/bluetooth/controller/include/ll_feat.h [hunk 1]
```diff
@@ -180,6 +186,7 @@
 /* Mask for host controlled features */
 #define LL_FEAT_HOST_BIT_MASK    0x100000000ULL
 
+/* Feature bits of this controller */
```
Comment (Thalley) [inline subsys/bluetooth/controller/include/ll_feat.h:189]: Looks like this comment got duplicated

**Review by Thalley — CHANGES_REQUESTED** [review]
Small request to remove duplicated comment

File: subsys/bluetooth/controller/hci/hci.c [hunk 2]
```diff
@@ -791,6 +791,20 @@ static void read_supported_commands(struct net_buf *buf, struct net_buf **evt)
...
+#if defined(CONFIG_BT_CTLR_DF_SCAN_CTE_RX)
+	rp->commands[39] |= BIT(7);
+#endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
+	rp->commands[40] |= BIT(4);
+#endif /* CONFIG_BT_CTLR_DF */
+
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
+	rp->commands[40] |= BIT(5);
+#endif /* CONFIG_BT_CTLR_SYNC_PERIODIC */
+
```
Comment (cvinayak) [inline subsys/bluetooth/controller/hci/hci.c:807]: Add comments as to which HCI commands are these, refer to others above. Also, have you used tabs or spaces? the leading alignments dont look like tabs of 8 spaces.

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/hci/hci.c (hunk 2 above)
Comment (Thalley) [reply subsys/bluetooth/controller/hci/hci.c:807]: Regarding the indentation, it seems like it is tabs, but Github has always been terrible at showing those correctly.

File: subsys/bluetooth/controller/include/ll_feat.h (hunk 1 above)
Comment (ppryga) [reply subsys/bluetooth/controller/include/ll_feat.h]: Thanks, duplicated comment removed.

File: subsys/bluetooth/controller/hci/hci.c (hunk 2 above)
Comment (ppryga) [reply subsys/bluetooth/controller/hci/hci.c]: Command description added.
Regarding the indentation, there are tabs in the code.

**Comment by ppryga** [comment]
@cvinayak @Thalley comments were addressed. Please take a look.

**Review by Thalley — APPROVED** [review]

**Review by cvinayak — APPROVED** [review]

**Pull request merged**